### PR TITLE
Feat: Update to 24.1.455

### PR DESCRIPTION
**Pull request by ludovicsteinbach**
- Update API specification from MI RC
- Bump version to 2.0.0
- Bump openapi-client-template to 2.0.3, to fix discriminator tests

Successful CI run on last commit https://github.com/ansys/grantami-serverapi-openapi/actions/runs/6801091574

### CODE REVIEW

File: ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py [hunk 1]
```diff
@@ -41,10 +47,13 @@
 from .api.schema___tables_api import SchemaTablesApi
 from .api.schema___units_api import SchemaUnitsApi
 from .api.search_api import SearchApi
+from .api.selection_searches_api import SelectionSearchesApi
 from .api.status_api import StatusApi
 
 # import models into sdk package
 from .models.folderguid_files_body import FolderguidFilesBody
+from .models.folderguid_files_body1 import FolderguidFilesBody1
+from .models.folderguid_files_body2 import FolderguidFilesBody2
```
Comment (ludovicsteinbach) [inline ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py:56]: Being investigated

Comment (Andy-Grigg) [reply ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py:56]: Is this a problem? I assume the code references the models correctly?

**Comment by Andy-Grigg** [comment]
Is the reason for the CI not running correctly due to the force push?

**Review by Andy-Grigg — APPROVED** [review]
I think this is fine, although I’m concerned about the CI not working.

We should review the definitions to make sure they’re correct.

**Comment by ludovicsteinbach** [comment]
> I think this is fine, although I’m concerned about the CI not working.
> 
> We should review the definitions to make sure they’re correct.

I think this due to https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow: since the changes are pushed by GH, it does not trigger other workflows.

File: ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py (hunk 1 above)
Comment (ludovicsteinbach) [reply ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py:56]: No. But I think the endpoints are not well documented, I need to have a better look

**Pull request merged**